.github/workflows/makefile.yml

```
@@ -67,11 +73,21 @@ jobs:
   macos:
     runs-on: macos-latest
 
+    env:
+      TRURL_JSON_IN: true
+
     steps:
       - uses: actions/checkout@v4
 
+      - name: Install Homebrew
+        id: install-homebrew
+        uses: Homebrew/actions/setup-homebrew@master
+      
+      - name: Install json-c
+        run: brew install json-c
+
       - name: make
-        run: make
+        run: make CFLAGS="-I /opt/homebrew/include" LDFLAGS="-L/opt/homebrew/lib"
```

```suggestion
        run: make CFLAGS="-I${brew --prefix}/include" LDFLAGS="-L${brew --prefix}/lib"
```
Default runners are ARM now, but let's adapt to any env automatically.

---

that broke something. Is it supposed to instead by $(brew --prefix) instead  of curly braces? we might be able to pull it from the install brew step as well.

---

Of course, my mistake, sorry!